src/docfx/build/Build.cs

```diff
@@ -76,14 +76,21 @@ public static async Task Run(string docsetPath, CommandLineOptions options, Repo
 
                 await ParallelUtility.ForEach(
                     docset.BuildScope.Where(doc => doc.ContentType != ContentType.TableOfContents),
-                    async (file, buildChild) => { monikersMap.TryAdd(file, await BuildOneFile(file, buildChild)); },
-                    ShouldBuildFile,
+                    async (file, buildChild) => { monikersMap.TryAdd(file, await BuildOneFile(file, buildChild, null, null)); },
+                    (file) => { return file.ContentType != ContentType.Resource && ShouldBuildFile(file); },
```

Comment:
>file.ContentType != ContentType.Resource [](start = 39, length = 40)

feels better to filter on the build scope above

Reply:
No, the build scope will not contains resource file, this is for filtering the `buildChild` action

---
In reply to: [237732440](https://github.com/dotnet/docfx/pull/3702#discussion_r237732440) [](ancestors = 237732440)